### ISSUE TITLE
Nethack does not install correctly

### ISSUE THREAD

**Issue by electr0sheep** [ArkSource/psphbb#16]
XMB says Corrupted Data

**Comment by ArkSource** [comment]
Could you confirm the app name?
Do you mean AirCrack-PSP?

**Comment by Northfear** [comment]
Nope. Just Nethack. The roguelike one. In the ports cathegory. Got the same problem unfortuanly.

**Comment by ArkSource** [comment]
Ok thanks heaps. I'll check it out. probably a simple typo causing corrupt download. Will let you know when it's ready to go.
@Northfear 
@electr0sheep
